homeassistant/components/tedee/config_flow.py

```diff
@@ -0,0 +1,57 @@
+"""Config flow for Tedee integration."""
+from typing import Any
+
+from pytedee_async import (
+    TedeeAuthException,
+    TedeeClient,
+    TedeeClientException,
+    TedeeLocalAuthException,
+)
+from pytedee_async.bridge import TedeeBridge
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_HOST
+from homeassistant.data_entry_flow import FlowResult
+
+from .const import CONF_LOCAL_ACCESS_TOKEN, DOMAIN, NAME
+
+
+class TedeeConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
+    """Handle a config flow for Tedee."""
+
+    async def async_step_user(
+        self, user_input: dict[str, Any] | None = None
+    ) -> FlowResult:
+        """Handle the initial step."""
+        errors: dict[str, str] = {}
+        local_bridge: TedeeBridge | None = None
+
+        if user_input is not None:
+            host = user_input.get(CONF_HOST, "")
```

Comment:
Schema says these values are required

```suggestion
            host = user_input[CONF_HOST]
```